gatsby-node.js

```diff
@@ -38,9 +37,22 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     const templateName = node.frontmatter.template || `default`
     const template = path.resolve(path.join('src/templates/', `${templateName}.js`))
     createPage({
-      path: node.frontmatter.path || node.parent.relativePath.replace(/.md$/,''),
+      path: node.frontmatter.path || node.fields.slug,
```

Comment:
Actually… Should this always be `slug` now?

Reply:
Yeah. Fixed! Initially I was worried it would affect the `pages` in the collection that do use paths, but apparently not

Reply:
Such confuse <_<